uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/request.controller.spec.js

```diff
@@ -676,6 +676,13 @@
           it('does not load contacts', function () {
             expect(controller.managedContacts.length).toEqual(0);
           });
+
+          it('has absence period matched with leave request dates', function () {
```

Comment:
a bit ambiguous, something like "selects the absence period that contains the leave request dates" would be better

Reply:
Done.